### PR TITLE
Reallocate IPs to accommodate extra external nets

### PR DESCRIPTION
**Pull request by Wasaac**
Squeezes networks comprised only of control plane hosts onto smaller subnets to accommodate extra `public_api` & `external_intranet` nets and expanding of `external_internet` net.

### CODE REVIEW

**Review by technowhizz — DISMISSED** [review]
LGTM

**Comment by Wasaac** [comment]
LGTM!

**Review by technowhizz — COMMENTED** [review]
LGTM

**Review by technowhizz — APPROVED** [review]

File: etc/kayobe/environments/production/network-allocation.yml [hunk 1]
```diff
@@ -7,32 +7,32 @@ external_ceph_net_ips:
...
-  controller2: 10.129.31.11
-  controller3: 10.129.31.12
-  controller4: 10.129.31.13
-external_internet_net_ips:
-  controller1: null
-  controller2: null
-  controller3: null
-  controller4: null
-external_intranet_net_ips:
+  controller1: 10.129.30.10
```
Comment (oneswig) [inline etc/kayobe/environments/production/network-allocation.yml:10]: Introducing a dedicated public API network is good, and will remove the need for binding control plane host IP addresses to the external_internet_net (or external_intranet_net).  We will instead need IP addresses on the external_api_net as below.  Elsewhere, references to binding the external_internet_net VLAN/subnet (eg, in network-interfaces) will need to be removed, as our external networks are being used solely for VM floating IPs and routers.  Neutron's problem now.

**Pull request merged**